### PR TITLE
Feature identity implementation

### PR DESCRIPTION
**Pull request by ZhevakSasha**

### CODE REVIEW

File: OnlineStore.MvcApplication/Controllers/LoginController.cs [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+{
+    /// <summary>
+    /// Login controller.
+    /// </summary>
+    public class LoginController : Controller
+    {
+        /// <summary>
+        /// Base url of api.
+        /// </summary>
+        private readonly string Baseurl = "https://localhost:44301/";
```
Comment (pavel-hiruts) [inline OnlineStore.MvcApplication/Controllers/LoginController.cs]: Move it to setting file

File: OnlineStore.MvcApplication/Controllers/RegisterController.cs [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+{
+    /// <summary>
+    /// Register controller.
+    /// </summary>
+    public class RegisterController : Controller
+    {
+        /// <summary>
+        /// Base url of api.
+        /// </summary>
+        private readonly string Baseurl = "https://localhost:44301/";
```
Comment (pavel-hiruts) [inline OnlineStore.MvcApplication/Controllers/RegisterController.cs]: Move it to settings file

File: OnlineStore.MvcApplication/Controllers/UserController.cs [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    {
+        /// <summary>
+        /// IHttpClientFactory.
+        /// </summary>
+        private readonly IHttpClientFactory _factory;
+
+        /// <summary>
+        /// Base url of api.
+        /// </summary>
+        private readonly string Baseurl = "https://localhost:44301/";
```
Comment (pavel-hiruts) [inline OnlineStore.MvcApplication/Controllers/UserController.cs]: Move it to settings file

**Review by pavel-hiruts — CHANGES_REQUESTED** [review]
All comments have to be fixed

File: OnlineStore.MvcApplication/Controllers/LoginController.cs (hunk 1 above)
Comment (ZhevakSasha) [reply OnlineStore.MvcApplication/Controllers/LoginController.cs]: Moved to appsettings

File: OnlineStore.MvcApplication/Controllers/RegisterController.cs (hunk 2 above)
Comment (ZhevakSasha) [reply OnlineStore.MvcApplication/Controllers/RegisterController.cs]: Moved to appsettings

File: OnlineStore.MvcApplication/Controllers/UserController.cs (hunk 3 above)
Comment (ZhevakSasha) [reply OnlineStore.MvcApplication/Controllers/UserController.cs]: Moved to appsettings

**Pull request merged**